sharding/nodesCoordinator/indexHashedNodesCoordinatorLite_test.go

```diff
@@ -0,0 +1,89 @@
+package nodesCoordinator
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestIndexHashedNodesCoordinator_SetNodesConfigFromValidatorsInfo(t *testing.T) {
+	t.Parallel()
+
+	arguments := createArguments()
+
+	ihgs, err := NewIndexHashedNodesCoordinator(arguments)
+	require.Nil(t, err)
```

Comment:
Is this check really necessary?
! This applies globally in this file

Reply:
not sure what should be the best practice here, while i'm testing if i change `createArguments` it might be handy to have this check also, but indeed, after everything is setup this is not really necessary